### ISSUE TITLE
ls

### ISSUE THREAD

**Issue by aaxu** [google/ci_edit#120]

**Comment by dschuyler** [comment]
How about if opening a directory on the command line goes directly into the Open (for that directory) in ci_edit

**Comment by aaxu** [comment]
Yeah that could work. I was thinking if we were to open a folder, it would count as opening a project. Then we could have a smart open feature, where it will recursively look through all subdirectories and index the files, so that if you search for a file, it will search for it in all directories. That way, you don't have to remember that file `x` has a path of a/b/c/d/x.

**Comment by aaxu** [comment]
Although it might not be too clear that you can actually create a new file from that "Open" window, since people are probably used to using a shortcut like CTRL_N or something to create a new file.
